sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/tests/Scenario/ServiceFabricManagedApplicationTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.ServiceFabricManagedClusters.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ServiceFabricManagedClusters.Tests
+{
+    internal class ServiceFabricManagedApplicationTests : ServiceFabricManagedClustersManagementTestBase
+    {
+        private ServiceFabricManagedApplicationCollection _appCollection;
+        private string _appTypeId;
+        public ServiceFabricManagedApplicationTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private async Task<ServiceFabricManagedApplicationTypeVersionResource> CreateTypeVersion(ServiceFabricManagedApplicationTypeResource appType)
+        {
+            string versionName = "1.0.0";
+            var data = new ServiceFabricManagedApplicationTypeVersionData(DefaultLocation)
+            {
+                AppPackageUri = new Uri("https://sfmcfortestapp.blob.core.windows.net/blobcontainer/VotingWebPkg.sfpkg")
```

Comment:
This file is public share, is this safe?

Reply:
Migrate this test from [track1 SDK](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicefabricmanagedclusters/Microsoft.Azure.Management.ServiceFabricManagedClusters/tests/Tests/ApplicationTests.cs#L22), this storage account has been deleted after test recording

Reply:
Then please change this uri to a dummy one.
And where did you ge this .sfpkg file? Please provide more info about how to get this file in comments.